web-services/model/src/main/java/datawave/webservice/query/model/ModelBean.java

```diff
@@ -205,7 +237,8 @@ else if (parts.length == 2)
     @GZIP
     @RolesAllowed({"Administrator", "JBossAdministrator"})
     @Interceptors(ResponseInterceptor.class)
-    public VoidResponse importModel(datawave.webservice.model.Model model, @QueryParam("modelTableName") String modelTableName) {
+    public VoidResponse importModel(datawave.webservice.model.Model model, @QueryParam("modelTableName") String modelTableName,
```

Comment:
As an additional optimization, can the importModel do the delete as well instead of returning a 412 response?  I always found that annoying personally.

Reply:
Lol! I had an earlier iteration with that exact change. But then I chickened out, because it seemingly violates traditional `POST` method semantics, and it felt like the original author(s) had those strict semantics in mind here.

But I agree with the annoyance factor here, and I could really go either way. Just let me know.

FWIW, from my own testing, I can say that a lot of the annoyance goes away when you don't have to wait minute(s) for an asynchronous cache refresh just to make it through to the finish line :-)

Reply:
In latest commit, I updated the `importModel` method to support an optional path parameter, i.e., `/import/{modelName}`,  in order to enable update/replace semantics